### DISCUSSION TITLE
Order of result on query

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#19546] [Q&A] [answered]
If I have a list of ids like this
```js
const ids = [
  'e552e394-0602-4d0e-ad23-77446de78500',
  '7396ba29-241a-47e4-b06c-cd8f0b285459',
  '11366a3f-6f9b-4b70-90f9-053a324ede08',
  '1e13cda1-6156-418b-b7d0-c754ae51f784'
];
```
And I load them with something like this
```js
const docs = await session
  .query({ indexName: 'SomeIndex' })
  .whereIn('id', ids)
  .selectFields(['id'])
  .all();
```
The order of the result is
```js
[
  '7396ba29-241a-47e4-b06c-cd8f0b285459',
  '1e13cda1-6156-418b-b7d0-c754ae51f784',
  '11366a3f-6f9b-4b70-90f9-053a324ede08',
  'e552e394-0602-4d0e-ad23-77446de78500'
]
```
I expected the order to be the same as the input? Is this an issue?

**Comment by karmeli87** [comment, marked as answer]
There is no order guarantee for querying without `OrderBy` usage.
In case you just want to load the entities with the given IDs there is an order guarantee if you use the `Load<T>(IEnumerable<string> ids)` method.
